### PR TITLE
Multi page future

### PR DESCRIPTION
**Pull request by jetexe**
i think we need to close pull #1 and start new here.

### CODE REVIEW

**Review by cjmaio — APPROVED** [review]

File: src/XlsxToCsv/XlsxToCsv.php [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
+<?php
 namespace XlsxToCsv;
 
 use \PclZip;
 
 class XlsxToCsv
 {
-    private $fileToConvert = null; 
+    private $fileToConvert = null;
+    public $sheetNomber    = 1;
```
Comment (cjmaio) [inline src/XlsxToCsv/XlsxToCsv.php]: I know it's small, but `$sheetNumber` should be the variable here.

File: src/XlsxToCsv/XlsxToCsv.php [hunk 2]
```diff
@@ -59,7 +60,7 @@ public function convert()
         $z->close( $filename );
 
         $dir = getcwd();
-        $filename = '/tmp/bin/xl/worksheets/sheet1.xml';
+        $filename = '/tmp/bin/xl/worksheets/sheet' . $this->sheetNomber . '.xml';
```
Comment (cjmaio) [inline src/XlsxToCsv/XlsxToCsv.php]: See above comment regarding spelling

**Review by cjmaio — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -7,5 +7,6 @@ An XLSX to CSV parser
 
 ```php
 $converter = new \XlsxToCsv\XlsxToCsv($filename);
+$converter->sheetNomber = 1;
```
Comment (cjmaio) [inline README.md]: Spelling question

**Review by cjmaio — CHANGES_REQUESTED** [review]

**Comment by cjmaio** [comment]
I think I grossly misused the review tool, turns out you don't need to submit per file. But anyway... just make this quick change, I'll get it tested, and then we can marge this in.

**Comment by jetexe** [comment]
i have add changes that you asked

**Pull request closed without merging**